code/modules/projectiles/projectile/bullets/junk.dm

```diff
@@ -0,0 +1,70 @@
+// Junk (Pipe Pistols and Pipeguns)
+
+/obj/projectile/bullet/junk
+	name = "junk bullet"
+	icon_state = "trashball"
+	damage = 30
+	embedding = list(embed_chance=15, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
+	var/bane_multiplier = 1.5
+	var/bane_added_damage = 0
+
+/obj/projectile/bullet/junk/shock/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/bane, mob_biotypes = MOB_ROBOTIC, target_type = /mob/living, damage_multiplier = bane_multiplier, added_damage = bane_added_damage)
+
+/obj/projectile/bullet/incendiary/fire/junk
+	name = "burning oil"
+	damage = 30
+	fire_stacks = 5
+	suppressed = SUPPRESSED_NONE
+
+/obj/projectile/bullet/junk/phasic
+	name = "junk phasic bullet"
+	projectile_phasing =  PASSTABLE | PASSGLASS | PASSGRILLE | PASSCLOSEDTURF | PASSMACHINE | PASSSTRUCTURE | PASSDOORS
+
+/obj/projectile/bullet/junk/shock
+	name = "bundle of live electrical parts"
+	icon_state = "tesla_projectile"
+	damage = 15
+	embedding = null
+	shrapnel_type = null
+	bane_multiplier = 2
+
+/obj/projectile/bullet/junk/shock/on_hit(atom/target, blocked = 0, pierce_hit)
+	. = ..()
+	if(isliving(target))
+		var/mob/living/victim = target
+		victim.electrocute_act(damage, src, siemens_coeff = 1, flags = SHOCK_NOSTUN)
+
+/obj/projectile/bullet/junk/hunter
+	name = "junk hunter bullet"
+
+/obj/projectile/bullet/junk/hunter/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/bane, mob_biotypes = MOB_BEAST, target_type = /mob/living, damage_multiplier = 0, added_damage = 50)
```

Comment:
Why doesn't this use the bane vars you put on the basetype?
Alternately, is it worth having those on the basetype if only two children use it?

Reply:
Oh I forgot to do that, shit.